commands/twitter.js

```diff
@@ -26,17 +26,21 @@ export class Twitter extends Command {
     });
 
     return new Promise((resolve, reject) => {
-      twitter.getTweet({id: tweet_id},
+      twitter.getTweet({id: tweet_id, include_entities: true},
         (error) => {
           console.log(`ERROR: Twitter Info - ${error}`);
           reject();
         },
         (success) => {
           let data = JSON.parse(success);
+          let text = data.text;
+          data.entities.urls.reverse().forEach(url => {
```

Comment:
I found the description "some or all matches of a pattern replaced by a replacement" worrisome and didn't bother testing how it actually worked lol